views/partials/head.ejs

```diff
@@ -6,6 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="/css/main.css">
+
+    <script type="module" src='https://unpkg.com/tesseract.js@v2.1.0/dist/tesseract.min.js'></script>
+    <script type="module" src="//unpkg.com/string-similarity/umd/string-similarity.min.js"></script>
```

Comment:
I believe this isn't the correct url? Other than that, nice!

Reply:
yes npm is weird 😄